include/pybind11/numpy.h

```diff
@@ -164,7 +164,7 @@ struct npy_api {
             NPY_ULONG_, NPY_ULONGLONG_, NPY_UINT_),
     };
 
-    using PyArray_Dims = struct {
```

Comment:
I'm just curious, when I saw this initially I thought it was something automatically applied, thinking "probably for a good reason". Was that not the case?

Reply:
It should have been automatically caught by the github actions tbh, but it wasn't. From modernize-use-using check